Sprint-1/errors/4.js

```diff
@@ -1,2 +1,3 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const HourClockTime12 = "20:53";
```

Comment:
Probably a CYF mistake, but did you notice the variable names (besides the syntax error) do not quite match the values assigned to them?

Reply:
I fixed the variables name